modules/ROOT/pages/manage-versions.adoc

```diff
@@ -0,0 +1,62 @@
+= Manage Versions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Manage the versions of an asset by viewing version detail, adding a new version, deleting a version, and changing the lifecycle state for a version.
+
+== View Version Detail
+You can view the following information about a version of an asset:
+
+- Name of the owner who published the asset
+- Date the version was published
+- Lifecycle status
+--
+
+To view the details of a version:
+
+. Click *Manage versions* in the expanded asset details pane.
+. Click the three dots menu and choose *Version detail* from the menu.
+-- 
+== Add New Version of an Asset
+
+If you have contributor or admin access to an asset, you can add a new version for an asset. The new version of the asset has the same name, icon, and description as the previous version. Any changes to these properties apply to all versions of the asset.
+
+. Click *Manage versions* in the expanded asset details pane.
+. Click *Add new version*.
+. (Required for APIs) Select a *Method* for the API specification.
+. (Required for APIs) From *File upload* click *Choose file*. For OAS, choose an OAS file with the .yaml or .json file types. For RAML, choose a RAML file with the .raml file type. Zip files are also accepted with the file to upload residing in the zip file’s root directory.
+. Specify the *Main file* if the uploaded file is a zip. Otherwise, Exchange uses the same file name as the file you upload. 
+. (Required) In *Version* , enter a new version for the asset.
+. (Required) In *the *API version*, enter the version of the API specification.
+The Version and API version values must agree. You can only choose minor version updates for a given API version. To set a major Version number, also increase the API version. If you choose minor or patch version updates, the API versions should not changed.
+
+. (Optional) Select a *Lifecycle state*. The default is *Stable*.
+. Click *Publish*
+
+
+== Delete a Version of an Asset
+
+You can delete versions of an asset. If you try to delete an asset version with dependent assets, Exchange shows a warning. Remove the dependencies and then delete the asset version.
+
+. Click *Manage versions* in the expanded asset details pane.
+. Click the three dots menu and choose *Delete version* from the menu.
+. Click *Delete version* from the confirmation dialog.
+
+
+== Change Lifecycle State for a Version of an Asset
+
+The lifecycle state for a version of an asset can be changed depending on the following state promotions:
+
+* `development` to `stable`
+* `stable` to `deprecated`
+* `deprecated` to `stable`
+
+If an asset in the development state depends on one or more other assets in the development state, Exchange does not allow promoting that asset to the stable state. All its dependencies must first be promoted to stable.
```

Comment:
```suggestion
If an asset in the development state depends on one or more other assets in the development state, Exchange does not promote that asset to the stable state. All of an asset's dependencies must first be promoted to a stable state before it can be promoted.
```